staging/src/k8s.io/apiserver/pkg/registry/rest/resttest/resttest.go

```diff
@@ -105,9 +105,6 @@ func (t *Tester) TestNamespace() string {
 // TestContext returns a namespaced context that will be used when making storage calls.
 // Namespace is determined by TestNamespace()
 func (t *Tester) TestContext() context.Context {
-	if t.clusterScope {
-		return genericapirequest.NewContext()
```

Comment:
should we be returning a context populated with NamespaceNone for cluster-scoped objects?

Reply:
Oh, that's what TestNamespace() returns for cluster scoped :)